app/views/layouts/application.html.slim

```diff
@@ -71,7 +71,7 @@ html
 
       / Main
       .container
-          button.btn.btn-success.pull-right onclick="show_unicorn();" Rainbows and Unicorns
+          / button.btn.btn-success.pull-right onclick="show_unicorn();" Rainbows and Unicorns
```

Comment:
FYI, we can kill this Rainbows and Unicorns thing....or make it smaller. It's kinda getting old as a joke.